package.json

```diff
@@ -112,6 +110,9 @@
     "wait-on": "^6.0.0",
     "yargs": "^17.2.1"
   },
+  "overrides": {
+    "lodash": "^4.17.21"
+  },
```

Comment:
Enforces ^4.17.21 of transient lodash dependency in npm@^8.0 to resolve 4 security vulnerabilities from previous versions.